src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -118,15 +119,17 @@ slots:
   subject_id:
     description: The ID of the subject of the mapping.
     range: EntityReference
-    required: true
     mappings:
     - owl:annotatedSource
     slot_uri: owl:annotatedSource
     examples:
       - value: HP:0009894
         description: The CURIE denoting the Human Phenotype Ontology concept of 'Thickened ears'
-  literal:
+  subject_literal:
     description: The literal being mapped
+    see_also:
+    - https://mapping-commons.github.io/sssom/sssom-profiles/
+    - https://github.com/mapping-commons/sssom/issues/234
     range: string
     required: true
```

Comment:
If you insist on having that separate slot (rather than reusing `subject_label`), you need to remove that `required`.